addons/im_livechat_mail_bot/models/mail_bot.py

```diff
@@ -12,18 +12,18 @@ def _get_answer(self, record, body, values, command):
         if self._is_bot_in_private_channel(record):
             if odoobot_state == "onboarding_attachement" and values.get("attachment_ids"):
                 self.env["mail.canned.response"].create({
-                    "source": "Thanks",
-                    "substitution": "Thanks for your feedback. Goodbye!",
-                    "description": "This is a temporary canned response to see how canned responses work.",
+                    "source": _("Thanks"),
+                    "substitution": _("Thanks for your feedback. Goodbye!"),
+                    "description": _("This is a temporary canned response to see how canned responses work."),
                 })
                 self.env.user.odoobot_failed = False
                 self.env.user.odoobot_state = "onboarding_canned"
                 return Markup(_("Wonderful! 😇<br/>Try typing %s to use canned responses. I've created a temporary one for you.", "<span class=\"o_odoobot_command\">:</span>"))
             elif odoobot_state == "onboarding_canned" and self.env.context.get("canned_response_ids"):
                 self.env["mail.canned.response"].search([
                     ("create_uid", "=", self.env.user.id),
-                    ("source", "=", "Thanks"),
-                    ("description", "=", "This is a temporary canned response to see how canned responses work."),
+                    ("source", "=", _("Thanks")),
+                    ("description", "=", _("This is a temporary canned response to see how canned responses work.")),
```

Comment:
What about inline comment to make sure both description should exactly match?
This looks obvious in the diff, and also the LOCs are relatively closed, but if someone change the wording of strings and only adapt one without the other, it looks likely that descriptions do not match and the `Thanks` is persistent rather than temporary.

Not a huge deal, you could ignore it if you feel this is fine as it is. Just wanted to point out this could happen and from my experience this could happen quite easily. If the sentence is changed in a stable version, ensuring the entry is updated once in POT file is fine, but the rewording could be in `master` and the POT diff during a freeze is too big to spot the mistake, resulting to the bug that is hardly fixable due to record being stored in DB